Client/LocalPlayer.cpp

```diff
@@ -85,7 +90,42 @@ void LocalPlayer::update() {
     _camera->set_position(_playerEntity->getState()->pos + _distance * _offset);
     _camera->Update();
 }
+void LocalPlayer::inputReadHandler()
+{
+	GamePadXbox* pad = new GamePadXbox(GamePadIndex_One);
 
+	while (1)
```

Comment:
This is a tight loop, and will completely hog the CPU on its thread. This won't directly impact us because our machines have multiple cores, but it is definitely not optimal. We probably want to handle the input firing / stick handling from the main loop instead of continuously. 

Maybe have `pad` accessible from Application.cpp, and move this function there (without the thread and while loop), or perhaps into InputManager? Weichen or Andy probably have the best idea of how this should be handled. I think we should probably consult with them before moving forward.